Cms_frontend/package.json

```diff
@@ -8,10 +8,12 @@
     "@testing-library/jest-dom": "^5.14.1",
```

Comment:
Looks good overall. There are a few ESLint flags but we can be lenient about it since this is just a placeholder anyway.